tc/external/detail/islpp-inl.h

```diff
@@ -38,10 +38,14 @@ inline isl::aff operator*(isl::aff A, isl::val V) {
   return V * A;
 }
 
+inline isl::aff operator/(isl::aff A, isl::val V) {
+  isl::aff T(isl::local_space(A.get_space().domain()), V);
```

Comment:
This is needlessly complicated.  Let me send out some patches in my queue.

Reply:
it would be great if we were aware of that queue somehow